source/rust_verify/src/verifier.rs

```diff
@@ -499,6 +499,7 @@ impl Verifier {
         bucket_id: &BucketId,
         reporter: &impl air::messages::Diagnostics,
         source_map: Option<&SourceMap>,
+        reports_in_line_order: &std::cell::RefCell<Option<Vec<(Message, MessageLevel)>>>,
```

Comment:
For consistency, can we use `*_source_order` rather than `*_line_order`?

Reply:
Also based on our conversation offline, I renamed these variables to `diagnostics_to_report`, to (hopefully) better communicate the intent.